src/ts/uimanager.ts

```diff
@@ -1061,11 +1072,9 @@ export namespace UIManager.Demo.Factory {
             new AirPlayToggleButton(),
             new CastToggleButton(),
             new VRToggleButton(),
-            new AudioTrackSelectBox({
-              cssClasses: ['my-audio-track-selection-demo'],
-            }),
-            new SubtitleSelectBox({
-              cssClasses: ['my-subtitle-selection-demo'],
+            new SettingsToggleButton({
+              settingsPanel: subtitleSettingsPanel,
+              cssClass: 'ui-subtitles-settings-toggle-button',
             }),
             new SettingsToggleButton({ settingsPanel: settingsPanel }),
```

Comment:
You need to hide all visible settingspanels when another panel is opened. Currently we can have both visible at the same time. You can achieve this via the `UIInstanceManager.onComponentShow` event:

```ts
// Keep track of shown SettingsPanels
const shownSettingsPanels: SettingsPanel[] = [];
uiManager.onComponentShow.subscribe((sender) => {
  if (sender instanceof SettingsPanel) {
    shownSettingsPanels.push(sender);
    sender.onComponentHide.subscribeOnce(() => ArrayUtils.remove(shownSettingsPanels, sender));
  }
});

// Hide all open SettingsPanels before opening this button's panel
// (We need to iterate a copy because hiding them will automatically remove themselves from the array
// due to the subscribeOnce above)
shownSettingsPanels.slice().forEach(settingsPanel => settingsPanel.hide());

mySettingsPanel.show();
```

(this code is not tested but should give you the idea)